VERSION

```diff
@@ -0,0 +1 @@
+0.1.2
```

Comment:
Can we make the version 1.0.0-rc10 ? Is that even valid in rust? I know actix-web has 3.0.0-alpha.2, so I'd be happy to tweak it too. This way it lines up with what I've been tagging in git already and the versions in build.opensuse.org.